### ISSUE TITLE
Getting infinitely repeated lines in the console ragarding failed to open some directory

### ISSUE THREAD

**Issue by sanjay23singh** [flutter/flutter#64728]
while launching my app, i got some repated lines in the debug console
I tried flutter run from the root of the project folder it produced the same lines again and again.

I/chatty  (29094): uid=10008(com.example.DarkSky) identical 65 lines
E/UIFirst (29094): failed to open /proc/29094/stuck_info
E/UIFirst (29094): failed to open /proc/29094/stuck_info
E/UIFirst (29094): failed to open /proc/29094/stuck_info
E/UIFirst (29094): failed to open /proc/29094/stuck_info

These are the lines which i am getting in infinite loop on running my app.
Also to be noted the app runs smoothly only the debug console repeatedly shows these error lines.

**Comment by darshankawar** [comment]
Hi @sanjay23singh,
Can you elaborate on this issue ? Which IDE you see these logs on ? 
Can you try `flutter pub cache repair` and then `flutter run` to see if it works for you ?
If problem persists, provide a minimal code that shows these logs.
Thanks.

**Comment by sanjay23singh** [comment]
@darshankawar 
i am using VS code.
I also tried flutter pub cache repair followed by flutter run but that doesn't helped.

Also, no additional code is needed for this error,even the starting counter increment code is also giving this error.
please look into this as i have projects in pending.
Thank u.

**Comment by darshankawar** [comment]
Similar to [64152](https://github.com/flutter/flutter/issues/64152)

**Comment by darshankawar** [comment]
@sanjay23singh are you able to run app or not at all ? What actions are you taking in the app that throws these logs ?

**Comment by sanjay23singh** [comment]
yeah my app runs fine, like i am able to increment the counter,but my console log keeps printing these lines.

**Comment by darshankawar** [comment]
@sanjay23singh please provide `flutter doctor -v`. Since you are able to run and perform actions in the app, I don't know how it's affecting your projects.

**Comment by sanjay23singh** [comment]
C:\Users\sanjay>flutter doctor -v
[√] Flutter (Channel stable, 1.20.2, on Microsoft Windows [Version 10.0.18363.1016], locale en-IN)
    • Flutter version 1.20.2 at C:\Users\sanjay\Documents\Not to be deleted\flutter
    • Framework revision bbfbf1770c (3 weeks ago), 2020-08-13 08:33:09 -0700
    • Engine revision 9d5b21729f
    • Dart version 2.9.1


[√] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at C:\Users\sanjay\AppData\Local\Android\sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: C:\Program Files\Android\Android Studio2\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)
    • All Android licenses accepted.

[√] Android Studio (version 3.6)
    • Android Studio at C:\Program Files\Android\Android Studio2
    • Flutter plugin version 45.1.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)

[√] VS Code, 64-bit edition (version 1.48.2)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 3.13.2

[√] Connected device (1 available)
    • RMX1801 (mobile) • 8d20cc3f • android-arm64 • Android 10 (API 29)

• No issues found!

This is my report of flutter doctor -v
Any file i create new or old,this error appears.
Also this error of mine is appearing after my android update to android 10

**Comment by darshankawar** [comment]
@sanjay23singh can you switch to latest master or latest dev and see if you get those logs there too ? 
Also, note that, they are not errors.

**Comment by sanjay23singh** [comment]
@darshankawar what do we mean by latest dev and latest master?

**Comment by darshankawar** [comment]
@sanjay23singh,
you seem to be on stable version of Futter. Try to switch to dev version or master version and see if you get the same issue.

**Comment by iBhavesh** [comment]
I am having the same issue


flutter doctor -v
```
[✓] Flutter (Channel stable, 1.20.2, on Linux, locale en_IN)
    • Flutter version 1.20.2 at /home/bhavesh/dev/flutter
    • Framework revision bbfbf1770c (4 weeks ago), 2020-08-13 08:33:09 -0700
    • Engine revision 9d5b21729f
    • Dart version 2.9.1

 
[✓] Android toolchain - develop for Android devices (Android SDK version
    30.0.1)
    • Android SDK at /home/bhavesh/dev/SDK/Android
    • Platform android-30, build-tools 30.0.1
    • Java binary at: /opt/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Android Studio (version 4.0)
    • Android Studio at /opt/android-studio
    • Flutter plugin version 48.1.2
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[✓] Connected device (1 available)
    • RMX1801 (mobile) • 2076193b • android-arm64 • Android 10 (API 29)

• No issues found!
```


flutter run --verbose
```
[ +170 ms] executing: [/home/bhavesh/dev/flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +43 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[        ] bbfbf1770cca2da7c82e887e4e4af910034800b6
[        ] executing: [/home/bhavesh/dev/flutter/] git tag --contains HEAD
[ +272 ms] Exit code 0 from: git tag --contains HEAD
[        ] 1.20.2
[  +11 ms] executing: [/home/bhavesh/dev/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[   +5 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [/home/bhavesh/dev/flutter/] git ls-remote --get-url origin
[   +5 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[  +72 ms] executing: [/home/bhavesh/dev/flutter/] git rev-parse --abbrev-ref HEAD
[   +5 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[  +52 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +2 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +16 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb devices -l
[  +47 ms] List of devices attached
           2076193b               device usb:1-1 product:RMX1801 model:RMX1801 device:RMX1801 transport_id:3
[   +8 ms] /home/bhavesh/dev/SDK/Android/platform-tools/adb -s 2076193b shell getprop
[ +109 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[   +3 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +1 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +283 ms] Generating /media/sharedSSD/Projects/flutter/expense_tracker/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java
[  +36 ms] ro.hardware = qcom
[  +90 ms] Starting incremental build...
[   +3 ms] Initializing file store
[  +15 ms] Skipping target: gen_localizations
[  +12 ms] complete
[   +7 ms] Launching lib/main.dart on RMX1801 in debug mode...
[   +8 ms] /home/bhavesh/dev/flutter/bin/cache/dart-sdk/bin/dart --disable-dart-dev
/home/bhavesh/dev/flutter/bin/cache/artifacts/engine/linux-x64/frontend_server.dart.snapshot --sdk-root
/home/bhavesh/dev/flutter/bin/cache/artifacts/engine/common/flutter_patched_sdk/ --incremental --target=flutter --debugger-module-names
-Ddart.developer.causal_async_stacks=true --output-dill /tmp/flutter_tools.GXCBIQ/flutter_tool.BEKMHZ/app.dill --packages .packages -Ddart.vm.profile=false
-Ddart.vm.product=false
--bytecode-options=source-positions,local-var-info,debugger-stops,instance-field-initializers,keep-unreachable-code,avoid-closure-call-instructions --enable-asserts
--track-widget-creation --filesystem-scheme org-dartlang-root --initialize-from-dill build/cache.dill.track.dill
[  +28 ms] executing: /home/bhavesh/dev/SDK/Android/build-tools/30.0.1/aapt dump xmltree
/media/sharedSSD/Projects/flutter/expense_tracker/build/app/outputs/flutter-apk/app.apk AndroidManifest.xml
[   +9 ms] Exit code 0 from: /home/bhavesh/dev/SDK/Android/build-tools/30.0.1/aapt dump xmltree
/media/sharedSSD/Projects/flutter/expense_tracker/build/app/outputs/flutter-apk/app.apk AndroidManifest.xml
[        ] N: android=http://schemas.android.com/apk/res/android
             E: manifest (line=2)
               A: android:versionCode(0x0101021b)=(type 0x10)0x1
               A: android:versionName(0x0101021c)="1.0.0" (Raw: "1.0.0")
               A: android:compileSdkVersion(0x01010572)=(type 0x10)0x1c
               A: android:compileSdkVersionCodename(0x01010573)="9" (Raw: "9")
               A: package="com.example.expense_tracker" (Raw: "com.example.expense_tracker")
               A: platformBuildVersionCode=(type 0x10)0x1c
               A: platformBuildVersionName=(type 0x10)0x9
               E: uses-sdk (line=7)
                 A: android:minSdkVersion(0x0101020c)=(type 0x10)0x10
                 A: android:targetSdkVersion(0x01010270)=(type 0x10)0x1c
               E: uses-permission (line=14)
                 A: android:name(0x01010003)="android.permission.INTERNET" (Raw: "android.permission.INTERNET")
               E: application (line=22)
                 A: android:label(0x01010001)="Expense Tracker" (Raw: "Expense Tracker")
                 A: android:icon(0x01010002)=@0x7f080000
                 A: android:name(0x01010003)="io.flutter.app.FlutterApplication" (Raw: "io.flutter.app.FlutterApplication")
                 A: android:debuggable(0x0101000f)=(type 0x12)0xffffffff
                 A: android:appComponentFactory(0x0101057a)="androidx.core.app.CoreComponentFactory" (Raw: "androidx.core.app.CoreComponentFactory")
                 E: activity (line=28)
                   A: android:theme(0x01010000)=@0x7f0a0000
                   A: android:name(0x01010003)="com.example.expense_tracker.MainActivity" (Raw: "com.example.expense_tracker.MainActivity")
                   A: android:launchMode(0x0101001d)=(type 0x10)0x1
                   A: android:configChanges(0x0101001f)=(type 0x11)0x40003fb4
                   A: android:windowSoftInputMode(0x0101022b)=(type 0x11)0x10
                   A: android:hardwareAccelerated(0x010102d3)=(type 0x12)0xffffffff
                   E: meta-data (line=42)
                     A: android:name(0x01010003)="io.flutter.embedding.android.NormalTheme" (Raw: "io.flutter.embedding.android.NormalTheme")
                     A: android:resource(0x01010025)=@0x7f0a0001
                   E: meta-data (line=52)
                     A: android:name(0x01010003)="io.flutter.embedding.android.SplashScreenDrawable" (Raw: "io.flutter.embedding.android.SplashScreenDrawable")
                     A: android:resource(0x01010025)=@0x7f040000
                   E: intent-filter (line=56)
                     E: action (line=57)
                       A: android:name(0x01010003)="android.intent.action.MAIN" (Raw: "android.intent.action.MAIN")
                     E: category (line=59)
                       A: android:name(0x01010003)="android.intent.category.LAUNCHER" (Raw: "android.intent.category.LAUNCHER")
                 E: meta-data (line=66)
                   A: android:name(0x01010003)="flutterEmbedding" (Raw: "flutterEmbedding")
                   A: android:value(0x01010024)=(type 0x10)0x2
[   +9 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb -s 2076193b shell -x logcat -v time -t 1
[  +75 ms] Exit code 0 from: /home/bhavesh/dev/SDK/Android/platform-tools/adb -s 2076193b shell -x logcat -v time -t 1
[        ] --------- beginning of main
           09-09 13:23:34.053 D/        ( 1593): charger_monitor /sys/class/power_supply/ac/online read fail -1
[   +5 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb -s 2076193b shell -x logcat -v time -t 1
[  +95 ms] Exit code 0 from: /home/bhavesh/dev/SDK/Android/platform-tools/adb -s 2076193b shell -x logcat -v time -t 1
[        ] --------- beginning of main
           09-09 13:23:34.053 D/        ( 1593): charger_monitor /sys/class/power_supply/ac/online read fail -1
[  +44 ms] <- compile package:expense_tracker/main.dart
[  +15 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb version
[   +9 ms] Android Debug Bridge version 1.0.41
           Version 30.0.3-6597393
           Installed as /home/bhavesh/dev/SDK/Android/platform-tools/adb
[   +2 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb start-server
[  +23 ms] Building APK
[  +31 ms] Running Gradle task 'assembleDebug'...
[   +3 ms] gradle.properties already sets `android.enableR8`
[   +5 ms] Using gradle from /media/sharedSSD/Projects/flutter/expense_tracker/android/gradlew.
[   +1 ms] /media/sharedSSD/Projects/flutter/expense_tracker/android/gradlew mode: 33279 rwxrwxrwx.
[  +16 ms] executing: /opt/android-studio/jre/bin/java -version
[ +100 ms] Exit code 0 from: /opt/android-studio/jre/bin/java -version
[        ] openjdk version "1.8.0_242-release"
           OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
           OpenJDK 64-Bit Server VM (build 25.242-b3-6222593, mixed mode)
[   +5 ms] executing: [/media/sharedSSD/Projects/flutter/expense_tracker/android/] /media/sharedSSD/Projects/flutter/expense_tracker/android/gradlew -Pverbose=true
-Ptarget-platform=android-arm64 -Ptarget=/media/sharedSSD/Projects/flutter/expense_tracker/lib/main.dart -Ptrack-widget-creation=true
-Pfilesystem-scheme=org-dartlang-root assembleDebug
[+1675 ms] > Task :app:compileFlutterBuildDebug UP-TO-DATE
[        ] > Task :app:packLibsflutterBuildDebug UP-TO-DATE
[        ] > Task :app:preBuild UP-TO-DATE
[        ] > Task :app:preDebugBuild UP-TO-DATE
[        ] > Task :app:compileDebugAidl NO-SOURCE
[        ] > Task :app:checkDebugManifest UP-TO-DATE
[        ] > Task :app:generateDebugBuildConfig UP-TO-DATE
[        ] > Task :app:compileDebugRenderscript NO-SOURCE
[  +72 ms] > Task :app:cleanMergeDebugAssets
[        ] > Task :app:mergeDebugShaders UP-TO-DATE
[        ] > Task :app:compileDebugShaders UP-TO-DATE
[        ] > Task :app:generateDebugAssets UP-TO-DATE
[        ] > Task :app:mergeDebugAssets
[ +697 ms] > Task :app:copyFlutterAssetsDebug
[        ] > Task :app:mainApkListPersistenceDebug UP-TO-DATE
[        ] > Task :app:generateDebugResValues UP-TO-DATE
[        ] > Task :app:generateDebugResources UP-TO-DATE
[  +99 ms] > Task :app:mergeDebugResources UP-TO-DATE
[        ] > Task :app:createDebugCompatibleScreenManifests UP-TO-DATE
[        ] > Task :app:processDebugManifest UP-TO-DATE
[        ] > Task :app:processDebugResources UP-TO-DATE
[        ] > Task :app:compileDebugKotlin UP-TO-DATE
[        ] > Task :app:javaPreCompileDebug UP-TO-DATE
[  +99 ms] > Task :app:compileDebugJavaWithJavac UP-TO-DATE
[        ] > Task :app:compileDebugSources UP-TO-DATE
[        ] > Task :app:processDebugJavaRes NO-SOURCE
[        ] > Task :app:mergeDebugJavaResource UP-TO-DATE
[        ] > Task :app:checkDebugDuplicateClasses UP-TO-DATE
[ +107 ms] > Task :app:desugarDebugFileDependencies UP-TO-DATE
[        ] > Task :app:mergeExtDexDebug UP-TO-DATE
[        ] > Task :app:transformClassesWithDexBuilderForDebug UP-TO-DATE
[        ] > Task :app:mergeDexDebug UP-TO-DATE
[        ] > Task :app:validateSigningDebug UP-TO-DATE
[  +92 ms] > Task :app:signingConfigWriterDebug UP-TO-DATE
[        ] > Task :app:mergeDebugJniLibFolders UP-TO-DATE
[        ] > Task :app:mergeDebugNativeLibs UP-TO-DATE
[        ] > Task :app:stripDebugDebugSymbols UP-TO-DATE
[        ] Compatible side by side NDK version was not found.
[        ] > Task :app:packageDebug UP-TO-DATE
[ +892 ms] > Task :app:assembleDebug
[        ] BUILD SUCCESSFUL in 3s
[        ] 31 actionable tasks: 4 executed, 27 up-to-date
[ +398 ms] Running Gradle task 'assembleDebug'... (completed in 4.3s)
[ +739 ms] calculateSha: LocalDirectory: '/media/sharedSSD/Projects/flutter/expense_tracker/build/app/outputs/flutter-apk'/app.apk
[ +105 ms] calculateSha: reading file took 104us
[+1014 ms] calculateSha: computing sha took 1014us
[  +10 ms] ✓ Built build/app/outputs/flutter-apk/app-debug.apk.
[   +6 ms] executing: /home/bhavesh/dev/SDK/Android/build-tools/30.0.1/aapt
dump xmltree
/media/sharedSSD/Projects/flutter/expense_tracker/build/app/outputs/flutter-
apk/app.apk AndroidManifest.xml
[  +14 ms] Exit code 0 from:
/home/bhavesh/dev/SDK/Android/build-tools/30.0.1/aapt dump xmltree
/media/sharedSSD/Projects/flutter/expense_tracker/build/app/outputs/flutter-
apk/app.apk AndroidManifest.xml
[        ] N: android=http://schemas.android.com/apk/res/android
             E: manifest (line=2)
               A: android:versionCode(0x0101021b)=(type 0x10)0x1
               A: android:versionName(0x0101021c)="1.0.0" (Raw: "1.0.0")
               A: android:compileSdkVersion(0x01010572)=(type 0x10)0x1c
               A: android:compileSdkVersionCodename(0x01010573)="9" (Raw:
               "9")
               A: package="com.example.expense_tracker" (Raw:
               "com.example.expense_tracker")
               A: platformBuildVersionCode=(type 0x10)0x1c
               A: platformBuildVersionName=(type 0x10)0x9
               E: uses-sdk (line=7)
                 A: android:minSdkVersion(0x0101020c)=(type 0x10)0x10
                 A: android:targetSdkVersion(0x01010270)=(type 0x10)0x1c
               E: uses-permission (line=14)
                 A: android:name(0x01010003)="android.permission.INTERNET"
                 (Raw: "android.permission.INTERNET")
               E: application (line=22)
                 A: android:label(0x01010001)="Expense Tracker" (Raw:
                 "Expense Tracker")
                 A: android:icon(0x01010002)=@0x7f080000
                 A:
                 android:name(0x01010003)="io.flutter.app.FlutterApplication
                 " (Raw: "io.flutter.app.FlutterApplication")
                 A: android:debuggable(0x0101000f)=(type 0x12)0xffffffff
                 A:
                 android:appComponentFactory(0x0101057a)="androidx.core.app.
                 CoreComponentFactory" (Raw:
                 "androidx.core.app.CoreComponentFactory")
                 E: activity (line=28)
                   A: android:theme(0x01010000)=@0x7f0a0000
                   A:
                   android:name(0x01010003)="com.example.expense_tracker.Mai
                   nActivity" (Raw:
                   "com.example.expense_tracker.MainActivity")
                   A: android:launchMode(0x0101001d)=(type 0x10)0x1
                   A: android:configChanges(0x0101001f)=(type
                   0x11)0x40003fb4
                   A: android:windowSoftInputMode(0x0101022b)=(type
                   0x11)0x10
                   A: android:hardwareAccelerated(0x010102d3)=(type
                   0x12)0xffffffff
                   E: meta-data (line=42)
                     A:
                     android:name(0x01010003)="io.flutter.embedding.android.
                     NormalTheme" (Raw:
                     "io.flutter.embedding.android.NormalTheme")
                     A: android:resource(0x01010025)=@0x7f0a0001
                   E: meta-data (line=52)
                     A:
                     android:name(0x01010003)="io.flutter.embedding.android.
                     SplashScreenDrawable" (Raw:
                     "io.flutter.embedding.android.SplashScreenDrawable")
                     A: android:resource(0x01010025)=@0x7f040000
                   E: intent-filter (line=56)
                     E: action (line=57)
                       A:
                       android:name(0x01010003)="android.intent.action.MAIN"
                       (Raw: "android.intent.action.MAIN")
                     E: category (line=59)
                       A:
                       android:name(0x01010003)="android.intent.category.LAU
                       NCHER" (Raw: "android.intent.category.LAUNCHER")
                 E: meta-data (line=66)
                   A: android:name(0x01010003)="flutterEmbedding" (Raw:
                   "flutterEmbedding")
                   A: android:value(0x01010024)=(type 0x10)0x2
[   +5 ms] Stopping app 'app.apk' on RMX1801.
[   +1 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb -s
2076193b shell am force-stop com.example.expense_tracker
[ +106 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb -s
2076193b shell pm list packages com.example.expense_tracker
[ +108 ms] Installing APK.
[   +4 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb
version
[  +12 ms] Android Debug Bridge version 1.0.41
           Version 30.0.3-6597393
           Installed as /home/bhavesh/dev/SDK/Android/platform-tools/adb
[        ] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb
start-server
[  +13 ms] Installing build/app/outputs/flutter-apk/app.apk...
[        ] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb -s
2076193b install -t -r
/media/sharedSSD/Projects/flutter/expense_tracker/build/app/outputs/flutter-
apk/app.apk
[+4960 ms] Performing Streamed Install
                    Success
[   +1 ms] Installing build/app/outputs/flutter-apk/app.apk... (completed in
5.0s)
[  +11 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb -s
2076193b shell echo -n 91b498c9d39ef6017deb8fce82e70b7377ff5283 >
/data/local/tmp/sky.com.example.expense_tracker.sha1
[  +99 ms] RMX1801 startApp
[  +11 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb -s
2076193b shell am start -a android.intent.action.RUN -f 0x20000000 --ez
enable-background-compilation true --ez enable-dart-profiling true --ez
enable-checked-mode true --ez verify-entry-points true
com.example.expense_tracker/com.example.expense_tracker.MainActivity
[ +152 ms] Starting: Intent { act=android.intent.action.RUN flg=0x20000000
cmp=com.example.expense_tracker/.MainActivity (has extras) }
[   +2 ms] Waiting for observatory port to be available...
[+2456 ms] Observatory URL on device: http://127.0.0.1:41266/Tch1Hs_Mt4M=/
[   +5 ms] executing: /home/bhavesh/dev/SDK/Android/platform-tools/adb -s
2076193b forward tcp:0 tcp:41266
[  +19 ms] 46525
[        ] Forwarded host port 46525 to device port 41266 for Observatory
[   +7 ms] Caching compiled dill
[ +311 ms] Connecting to service protocol:
http://127.0.0.1:46525/Tch1Hs_Mt4M=/
[ +400 ms] Successfully connected to service protocol:
http://127.0.0.1:46525/Tch1Hs_Mt4M=/
[        ] Waiting for RMX1801 to report its views...
[  +10 ms] Waiting for RMX1801 to report its views... (completed in 9ms)
[  +14 ms] DevFS: Creating new filesystem on the device (null)
[  +89 ms] DevFS: Created new filesystem on the device
(file:///data/user/0/com.example.expense_tracker/code_cache/expense_trackerA
YZTCD/expense_tracker/)
[   +9 ms] Updating assets
[ +135 ms] Syncing files to device RMX1801...
[   +3 ms] Scanning asset files
[   +4 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[        ] <- recompile package:expense_tracker/main.dart
246b2556-6fdc-45a5-a25a-0cceefc66350
[        ] <- 246b2556-6fdc-45a5-a25a-0cceefc66350
[  +84 ms] Updating files
[ +201 ms] DevFS: Sync finished
[        ] Syncing files to device RMX1801... (completed in 295ms)
[        ] Synced 0.9MB.
[   +1 ms] <- accept
[   +4 ms] Connected to _flutterView/0x726c823020.
[   +1 ms] Flutter run key commands.
[   +1 ms] r Hot reload. 🔥🔥🔥
[   +1 ms] R Hot restart.
[        ] h Repeat this help message.
[        ] d Detach (terminate "flutter run" but leave application running).
[        ] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ] An Observatory debugger and profiler on RMX1801 is available at:
           http://127.0.0.1:46525/Tch1Hs_Mt4M=/
[ +162 ms] I/flutter (19824): Wed
[   +1 ms] I/flutter (19824): 0.0
[   +1 ms] I/flutter (19824): Tue
[        ] I/flutter (19824): 0.0
[        ] I/flutter (19824): Mon
[        ] I/flutter (19824): 0.0
[        ] I/flutter (19824): Sun
[        ] I/flutter (19824): 0.0
[        ] I/flutter (19824): Sat
[        ] I/flutter (19824): 0.0
[        ] I/flutter (19824): Fri
[        ] I/flutter (19824): 0.0
[        ] I/flutter (19824): Thu
[        ] I/flutter (19824): 0.0
[ +939 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
[ +102 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
[  +10 ms] I/expense_tracke(19824): ProcessProfilingInfo new_methods=464 is
saved saved_to_disk=1 resolve_classes_delay=5000
[  +13 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
[  +29 ms] I/chatty  (19824): uid=10344(com.example.expense_tracker)
identical 1 line
[   +1 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
[   +5 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
[  +13 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
[ +426 ms] I/chatty  (19824): uid=10344(com.example.expense_tracker)
identical 48 lines
[   +2 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
[   +5 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
[  +66 ms] I/chatty  (19824): uid=10344(com.example.expense_tracker)
identical 8 lines
[   +2 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
[  +76 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
[ +307 ms] E/UIFirst (19824): failed to open /proc/19824/stuck_info, No such
file or directory
```

**Comment by TahaTesser** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.

**Comment by iBhavesh** [comment]
I don't really know the reason but I think its something related to realme UI, on other devices the log is clear but on the realme phone with realme ui log is noisy

**Comment by sanjay23singh** [comment]
@iBhavesh  that might be a reason i used to think as other devices worked fine.
but one thing that doesn't allows me to believe in that it is only occuring only after i updated to android 10 in my realme device.
before that it was not showing any logs.

Maybe that a problem related to android 10 upgrade.

**Comment by iBhavesh** [comment]
@sanjay23singh With android 10 realme users got realme ui maybe there is some bug in that upgrade, i tried on other android 10 devices worked fine for me, but on my realme 2 pro after realme ui upgrade log is noisy, for now i am just filtering the vs code console to not include the proc lines

**Comment by sanjay23singh** [comment]
@iBhavesh how are you filtering the console can u tell me?

**Comment by iBhavesh** [comment]
There's a field for filter in the debug console.![IMG20201015164738.jpg](https://user-images.githubusercontent.com/34792643/96116646-6e51b100-0f06-11eb-9c5f-b832366f5ed6.jpg)

Try this for now
